providers/openfeature-provider-ofrep/pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "openfeature-provider-ofrep"
-version = "0.1.0"
```

Comment:
You don't need to bump this manually. It's handled automatically by our release process.